neo/tools/common/MaskEdit.cpp

```diff
@@ -60,7 +60,7 @@ LRESULT CALLBACK MaskEdit_WndProc ( HWND hWnd, UINT msg, WPARAM wParam, LPARAM l
 
 		case WM_DESTROY:
 			delete edit;
-			SetWindowLong ( hWnd, GWL_WNDPROC, (LONG)wndproc );
+			SetWindowLongPtr ( hWnd, GWLP_WNDPROC, (LONG)wndproc );
```

Comment:
Missed this when merging this: pretty sure the `(LONG)` cast should be `(LONG_PTR)` instead.
There's more similar issues and there's also compiler warnings for it.
@raynorpat Can you look at your changes again and try to fix those issues?